app/Models/Post.php

```diff
@@ -157,7 +157,7 @@ public function toBlinkPayload()
         return [
             'id' => $this->id,
             'signup_id' => $this->signup_id,
-            'quantity' => $this->signup->quantity,
+            'quantity' => is_null($this->quantity) ? $this->signup->quantity : $this->quantity,
```

Comment:
I think we want to return `null` if we have `null` stored on the post. So we should really just return what is on the post